docs.feldera.com/docs/pipelines/transactions.md

```diff
@@ -0,0 +1,256 @@
+the **backfill** problem. While backfill can be performed in continuous mode,
+this is likely to lead to two issues:
+
+* **Performance:** computing all intermediate updates can be more expensive than
+  computing the cumulative update. The exact performance difference depends on
+  your SQL queries and can be very significant in some cases.
+
+* **Atomicity:** in continuous mode the pipeline produces a stream of intermediate
+  updates, which often cancel each other out.  Consider for instance a `COUNT(*)`
+  query whose output changes any time new records are ingested.  These updates
+  expose intermediate computation results to downstream systems while placing
+  excessive load on the data sinks.
+
+**Transactions** solve these challenges by allowing input updates to be grouped into
+a single batch and processed atomically. In transactional mode,
+Feldera still executes SQL queries incrementally and produces a delta of changes
+to output views. The key difference is in who controls the batch size: instead of the
+engine automatically  deciding chunk boundaries, the scope of
+each transaction is explicitly specified by the user.
+
+The following table summarizes the two modes:
+
+|                               | Continuous Mode                                                                                                                                | Transactions                                                                                                                                                         |
+|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **When to use**               | <p>Process real-time inputs with low latency</p>                                                                                               | <p>Efficiently process bulk data</p>                                                                                                                                 |
+| **When query evaluation is triggered** | <p>When an input chunk is ingested</p>                                                                                          | <p>On transaction commit</p>                                                                                                                                         |
+| **Strong consistency**        | <p>Yes: after processing an input chunk, the contents of all views matches all the inputs received so far.</p>                                 | <p>Yes: after committing a transaction the contents of all matches all the inputs received so far, including inputs received as part of the transaction.</p>         |
+
+## Transaction API
```

Comment:
should say someplace that these differ from standard database transactions in that they are not a concurrency control mechanism